backend/src/utils/log/logger.ts

```diff
@@ -0,0 +1,25 @@
+import winston from 'winston';
+
+import { PROD, QA } from '../../config/constants';
```

Comment:
You should better use absolute path import here. It's not really correct to use relative ones when you need to get back to src level to import something from another src-level folder.
```suggestion
import { PROD, QA } from 'config/constants';
```